### PR TITLE
Compliance

### PR DESCRIPTION
**Pull request by duncan-rocket**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ npm install
 Build and preview Astro on a local server. By default Astro will be create a server running on port 8080 and can be accessed via http://localhost:8080 check your build process to confirm the server location.
 
 ```
-npm run start
```
Comment (duncan-rocket) [inline README.md:39]: d'oh 🤦

File: _content/design-guidelines/compliance.md [hunk 2]
```diff
@@ -2,93 +2,7 @@
...
-			
-	17. [Tabs](/components/tabs)
-		1. Tabs shall be arranged in a single row.
-		2. Tabs shall use title-style capitalization.
-		
-	18. [Toggle](/components/toggle)
-		1. Toggle shall be used when the action takes immediate effect.
-		
-
+# Compliance
```
Comment (duncan-rocket) [inline _content/design-guidelines/compliance.md:8]: Should this be **EGS Compliance** still @rocketjeff

File: css/chapters.css [hunk 3]
```diff
@@ -16,16 +16,16 @@ main {
 /* main :not(figure):not(img):not(table) {
 	max-width: var(--content-max-text-width, 40em);
 } */
```
Comment (duncan-rocket) [inline css/chapters.css:18]: If this is commented out, should it be removed.

File: _content/design-guidelines/compliance.md (hunk 2 above)
Comment (duncan-rocket) [reply _content/design-guidelines/compliance.md:8]: Just confirmed with Jeff, this should be EGS Compliance

File: _content/design-guidelines/compliance.md [hunk 4]
```diff
@@ -2,93 +2,7 @@
 tags: resources
 path: /design-guidelines/compliance
 date: Last Modified
-layout: interior.template.njk
+layout: compliance.template.njk
 title: Compliance
```
Comment (duncan-rocket) [inline _content/design-guidelines/compliance.md:6]: Please change title to EGS Compliance

File: .eleventy.js [hunk 5]
```diff
@@ -22,31 +25,26 @@ module.exports = function(eleventyConfig) {
     .use(implicitFigures, {
       figcaption: true
     })
+    .use(markdownItFigure, {})
     .use(markdownItContainer, "note")
     .use(markdownItContainer, "caution")
     .use(markdownItContainer, "col")
     .use(markdownItContainer, "two-col")
-    .use(markdownItContainer, "three-col");
+    .use(markdownItContainer, "three-col")
```
Comment (sknep) [inline .eleventy.js:33]: missing a semicolon here

**Pull request closed without merging**